Regression/RegressionTree/src/test/java/org/tribuo/regression/rtree/TestCARTJointRegressionTrainer.java

```diff
@@ -36,13 +39,14 @@
 
     private static final CARTJointRegressionTrainer t = new CARTJointRegressionTrainer();
     private static final CARTJointRegressionTrainer normedt = new CARTJointRegressionTrainer(Integer.MAX_VALUE,true);
+    private static final CARTJointRegressionTrainer randomt = new CARTJointRegressionTrainer(Integer.MAX_VALUE, 5, 0.0f,
+            0.75f, true, new MeanSquaredError(), false, Trainer.DEFAULT_SEED);
 
-    public void testJointRegressionTree(Pair<Dataset<Regressor>,Dataset<Regressor>> p) {
-        TreeModel<Regressor> m = t.train(p.getA());
+    public void testJointRegressionTree(Pair<Dataset<Regressor>,Dataset<Regressor>> p, AbstractCARTTrainer<Regressor> trainer) {
```

Comment:
Similar to the classification tests, I'd prefer it if the sharp `CARTJointRegressionTrainer` is used rather than `AbstractCARTTrainer<Regressor>` unless you're sharing the tests across both types of regression tree trainer.